tests/h/services/delete_user_test.py

```diff
@@ -73,14 +73,17 @@ def test_delete_user_removes_groups_if_no_collaborators(
         db_session.flush()
         assert sqlalchemy.inspect(group).was_deleted
 
-    def test_delete_user_fails_if_groups_have_collaborators(
+    def test_creator_is_unassigned_if_groups_have_collaborators(
```

Comment:
I wonder if this function name should explicitly reference `None`, as "unassigned" is a higher-level concept (it's concretely setting `creator` to `None`).